src/cloud-api-adaptor/libvirt/config_libvirt.sh

```diff
@@ -118,3 +121,11 @@ rm -f libvirt.properties
 echo "libvirt_uri=\"qemu+ssh://${USER}@${IP}/system?no_verify=1\"" >> libvirt.properties
 echo "libvirt_ssh_key_file=\"id_rsa\"" >> libvirt.properties
 echo "CLUSTER_NAME=\"peer-pods\"" >> libvirt.properties
+
+# switch to the appropriate e2e test and add configs to libvirt.properties as needed
+case $TEST_E2E_SECURE_COMMS in
+
+  *)
+    echo "processing none"
+    ;;
+esac
```

Comment:
Do we need this debug here given that it is echo'd above?

Reply:
I think it should stay here as it is the basis of what we will do next - e.g.:
```
# switch to the appropriate e2e test and add configs to libvirt.properties as needed
case $TEST_E2E_SECURE_COMMS in

  "withoutKbs")
    echo "processing withoutKbs"
    echo "SECURE_COMMS=\"true\"" >> libvirt.properties
    echo "SECURE_COMMS_KBS_ADDR=\"false\"" >> libvirt.properties
    echo "INITDATA=\"\"" >> libvirt.properties
    ;;

  *)
    echo "processing none"
    ;;
esac
```

It maybe better to remove the earlier `echo "SECURE_COMMS is ${TEST_E2E_SECURE_COMMS}"` if it feels we are putting too much output.

Reply:
Why can't you add that in when you start using it? At the moment it's not doing anything useful?

Reply:
Myabe the right thing to do is to spell this out more clearly by adding `echo "SECURE_COMMS=\"false\"" >> libvirt.properties`

So we will have:
```
# switch to the appropriate e2e test and add configs to libvirt.properties as needed
case $TEST_E2E_SECURE_COMMS in

  *)
    echo "processing none"
    echo "SECURE_COMMS=\"false\"" >> libvirt.properties
    ;;
esac
```

Will this do?

Reply:
I think that's okay, it won't be used by anything, so I still don't understand the need for it in this PR, but 🤷